### ISSUE TITLE
Invoice Email für Ernährungskurs

### ISSUE THREAD

**Issue by ursulafeu** [viszerale-therapie/simple-courses#157]
Lieber Andreas,
können wir für den einen Kurs Physiologische Ernährung ein eigenes Invoice Email anlegen?

Soll so sein - da steht weniger drin als bei den anderen:

${customer.full_salutation} ${customer.lastname}!

Anbei möchten wir die Kursinformationen und die Rechnung für den Kurs „${course.title}" zusenden.

Daten: ${course.startDate} - ${course.endDate}
Ort: ${location.name}
Kurszeiten: täglich 9:00–18:00, letzter Tag 8:30–17:30
Bei versäumen von mehr als 4 Stunden, muss der Kurstag kostenpflichtig (135€) nachgeholt werden, um eine Kursbestätigung zu bekommen.

Die Liste der TeilnehmerInnen zur Bildung der Fahrgemeinschaften wird Ihnen 2-3 Wochen vor Kursbeginn automatisch zugesandt, wenn Sie Ihre Kontaktdaten dazu frei gegeben haben.

Falls die Kursgebühr noch nicht einbezahlt wurde, bitten wir diese bis ${invoice.weeks} Wochen vor Kursbeginn (${invoice.payDate}) zu begleichen.
Bei jeder Umbuchung bzw. Stornierung fallen Bearbeitungs - bzw. Stornierungsgebühren an.
Allgemeine Geschäfts - und Stornobedingungen.

Wir freuen uns auf Sie!

Mit freundlichen Grüßen
Schule für Viszerale Therapie

**Comment by Andreas-Schoenefeldt** [comment]
Ok, ich würd hier einen mechanismus einbauen, für den man emails in Abhängigkeit zum kurstyp Überschreiben kann.

**Comment by Andreas-Schoenefeldt** [comment]
Nein, das meiste haben wir eh schon für die Automatische PDF erstellung.

**Comment by ursulafeu** [comment]
ja super, dann bitte her damit :)
